framework/base/View.php

```
@@ -380,8 +372,8 @@ public function renderPhpFile($_file_, $_params_ = [])
      */
     public function renderDynamic($statements)
     {
-        if (!empty($this->cacheStack)) {
-            $n = count($this->dynamicPlaceholders);
+        if (!empty($this->_dynamicContents)) {
+            $n = count($this->_dynamicPlaceholders);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $n. Configured minimum length is 3. (undefined)

(ShortVariable)